docs/topics_5/web-updating-an-analysis.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='updating_analysis_{context}']
+= Updating the analysis configuration
+
+You can update an analysis configuration, for example, with a different transformation target, advanced option, or a custom rule. Then you can run the updated analysis in your project.
+
+.Analysis configuration screen
+image::web-analysis-configuration.png[Analysis configuration screen]
+
+.Procedure
+
+. In the {WebName}, click *Analysis configuration*.
+. Select a *Project*.
+. Click the appropriate tabs - *General*, *Packages*, *Custom rules*, *Custom labels*, or *Adavanced options* - and make your changes.
```

Comment:
I commented on this yesterday. Don't quote tabs or tab orders and no dashes as punctuation